csm/conf/uds.py

```diff
@@ -72,15 +72,13 @@ def remove_csm_config(cls):
     @staticmethod
     def generate_haproxy_frontend_config():
         cluster_ip = SaltWrappers.get_salt_call('pillar.get', 'cluster:cluster_ip')
-        mgmt_vip = SaltWrappers.get_salt_call('pillar.get', 'cluster:mgmt_vip')
         return f"""\
 frontend uds-frontend
     mode tcp
     option tcplog
     bind 127.0.0.1:5000
     bind ::1:5000
     bind {cluster_ip}:5000
```

Comment:
We need to confirm that the haproxy should still start even if cluster IP is not available. (just like mgmt_vip issue)
if you look at - https://github.com/Seagate/cortx-prvsnr/blob/cortx-1.0/srv/components/ha/haproxy/files/haproxy.cfg, There is a handling by provisioner for S3 frontend section - which binds to cluster IP only if it is available
**{%- if cluster_ip %}
 bind {{ cluster_ip }}:80
 bind {{ cluster_ip }}:443 ssl crt /etc/ssl/stx/stx.pem
{%- endif %}**
Can you please check if we can do a similar handling.

Reply:
@AjinkyaDhumal Made the necessary changes please have a look.